operator/backupcontroller/executor.go

```diff
@@ -262,16 +263,17 @@ func (b *BackupExecutor) startBackup(ctx context.Context) error {
 			}
 			// each job sleeps for index seconds to avoid concurrent restic repository creation. Not the prettiest way but it works and a repository
 			// is created only once usually.
-			if index > 0 {
+			if name == "prebackup" || index != 0 {
```

Comment:
This fixes another small but unrelated issue.

If there's a PVC backup AND prebackup commands, it can happen that both want to `restic init` at the same time, which will corrupt the repo.